crates/fuzzing/src/oracles/diff_wasmi.rs

```diff
@@ -141,35 +129,42 @@ impl DiffInstance for WasmiInstance {
         arguments: &[DiffValue],
         result_tys: &[DiffValueType],
     ) -> Result<Option<Vec<DiffValue>>> {
-        let function = match self
+        let function = self
             .instance
             .get_export(&self.store, function_name)
-            .unwrap()
-        {
-            wasmi::Extern::Func(f) => f,
-            _ => unreachable!(),
-        };
+            .and_then(wasmi::Extern::into_func)
+            .unwrap();
```

Comment:
I'd suggest the same multi-unwrap pattern here too:

```suggestion
            .unwrap()
            .into_func()
            .unwrap();
```